Automattic/wc-calypso-bridge
# Convert plugins page to a WC page

### Changes proposed in this Pull Request:



Closes #1275  .



This PR converts `/plugins` (WP Core page) to a WC page to record tracks automatically and make the UI consistent.





### How to test the changes in this Pull Request:

I wasn't able to fully test the visual changes locally since we miss some CSS that are only available on a WPCOM site. Please use your WoA site and transfer the files in order to test.

1. Prepare a WoA site.
2. Checkout this branch in your WoA site.
3. Go to your WoA site.
4. Confirm `Plugins` menu is rendered.
5. Click `Plugins` menu.
6. Confirm the page is rendered correctly.
7. Open your browser console and execute `window.localStorage.setItem('debug', 'wc-admin:*')` and make sure to change log level to verbose.
8. Click buttons and links and make sure tracks are fired.
9. Confirm `wcadmin_page_view` track gets fired.
10. Click `Network` tab and search for `t.gif?path=plugins-upgrade` and click it.
11. Confirm `_en` parameter value is `wcadmin_page_view`




### Other information:




### FOR PR REVIEWER ONLY:
